app/src/main/java/com/hedvig/app/feature/webonboarding/WebOnboardingActivity.kt

```diff
@@ -93,7 +95,11 @@ class WebOnboardingActivity : BaseActivity(R.layout.activity_web_onboarding) {
                 else -> "no/"
             }
 
-            webOnboarding.loadUrl("${BuildConfig.WEB_BASE_URL}${localePath}new-member?variation=android#token=${encodedToken}")
+            when (marketProvider.market) {
+                Market.NO -> webOnboarding.loadUrl("${BuildConfig.WEB_BASE_URL}${webPath?.substring(1)}/start?variation=android#token=${encodedToken}")
```

Comment:
```suggestion
                Market.NO -> webOnboarding.loadUrl("${BuildConfig.WEB_BASE_URL}${webPath?.substring(1)}/start?variation=android#token=${encodedToken}")
```

How come we need to use `substring` here? 🤔

Reply:
I accidentally put a `/` in the beginning of the string in embark but there was already one in `BuildConfig.WEB_BASE_URL` ill fix that before merging